tests/config/test_emission_factors.py

```diff
@@ -9,7 +9,7 @@
 
 def test_all_emission_factors():
```

Comment:
You could just turn this into a massive snapshot test if you want.

Syrupy will handle it just fine.

Reply:
that's fair